src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/RefreshTest.java

```diff
@@ -0,0 +1,24 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+import ancaMarian.tests.tema8.pageObjects.RefreshPage;
```

Comment:
Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.RefreshPage;`